packages/widget/src/components/AlertSection.tsx

```diff
@@ -0,0 +1,23 @@
+import { styled } from '@mui/material/styles';
+import { Alert } from '@mui/material';
+
+export const AlertSection = styled(Alert)(({ theme, severity }) => ({
+  borderRadius: theme.shape.borderRadius,
+  backgroundColor:
+    severity === 'info'
+      ? theme.palette.mode === 'light'
+        ? theme.palette.grey[200]
+        : theme.palette.grey[800]
+      : 'transparent',
+  color: theme.palette.text.primary,
+  '.MuiAlert-icon': {
+    color:
+      severity === 'error'
+        ? theme.palette.mode === 'light'
+          ? theme.palette.error.light
+          : theme.palette.error.dark
+        : theme.palette.mode === 'light'
+          ? theme.palette.grey[700]
+          : theme.palette.grey[300],
+  },
+}));
```

Comment:
For some reason in Figma we have `info` state displayed as grey, but it should be blue for consistency with other existing states. Please take a look at `InfoMessageCard` components inside `components/GasMessage/GasMessage.style.ts`. We should make it reusable I think.